docs/the-new-architecture/use-app-template.md

```diff
@@ -3,5 +3,61 @@ id: use-app-template
 title: Using the New App Template
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+This page will help you create a new React Native app that uses the new architecture.
+
+## Installing dependencies
+
+Start by setting up your development environment as described in [Getting Started](getting-started.md), under "React Native CLI Quickstart". Stop when you reach "React Native Command Line Interface", and resume here.
+
+<h3>React Native Command Line Interface</h3>
+
+React Native has a built-in command line interface. Rather than install and manage a specific version of the CLI globally, we recommend you access the current version at runtime using `npx`, which ships with Node.js. With `npx react-native <command>`, the current stable version of the CLI will be downloaded and executed at the time the command is run.
+
+<h2>Creating a new application</h2>
+
+> If you previously installed a global `react-native-cli` package, please remove it as it may cause unexpected issues.
+
+React Native has a built-in command line interface, which you can use to generate a new project. You can access it without installing anything globally using `npx`, which ships with Node.js. Let's create a new React Native project called "AwesomeProject":
+
+```shell
+npx react-native init AwesomeProject
+```
+
+## Configuring iOS
+
+To configure iOS to use the new architecture, navigate to the `ios` directory and run the following:
+
+```shell
+# from `ios` directory
+RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
+```
+
+Then build and run the app as usual:
+
+```shell
+npx react-native run-ios
+```
+
+### Troubleshooting
+
+If you see a build failure, there may be cached files from a previous build with the old architecture. Clean the build cache and try again:
+
+```shell
+# from `ios` directory
+xcodebuild clean
+RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
+```
+
+## Configuring Android
```

Comment:
I did run into a problem here, but don't know the solution yet, so troubleshooting section to come.

The problem is that on attempting to `run-android` with the new architecture enabled on an M1 laptop, I get:

```* What went wrong:
Execution failed for task ':ReactAndroid:configureNdkBuildDebug'.
...
  ERROR: Unknown host CPU architecture: arm64
```

which looks like an M1-related issue, despite my attempts to use Zulu as the JDK (which appears to have actually been used).

Reply:
Discussed offline: there is currently a problem in invoking the NDK on M1s. The current workaround is listed here: https://reactnative.dev/docs/new-architecture-troubleshooting#android-ndk-and-mac-with-m1-apple-silicon-cpus (I would link against it).

I hope to be able to include this: 
- https://github.com/facebook/react-native/pull/33588

inside a point release so we won't need a workaround anymore

Reply:
Closing as this should be solved with 0.68.1 👍